<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gjenkjenning_send</class>
 <widget class="QWidget" name="gjenkjenning_send">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="side1">
      <widget class="QLabel" name="labelFrame">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>10</y>
         <width>791</width>
         <height>471</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="labelBarcodeData">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>570</y>
         <width>361</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="labelSavingImage">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>640</y>
         <width>361</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonTakePicture">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>570</y>
         <width>311</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ta bilde</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonScanBarcode">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>500</y>
         <width>311</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Skanne barkode</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>640</y>
         <width>311</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Avbryt</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonStartKamera">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>500</y>
         <width>361</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Start kamera</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
